### PR TITLE
Admin Overview

### PR DESCRIPTION
**Pull request by mxmlnwbr**

### CODE REVIEW

**Comment by mxmlnwbr** [comment]
@rschlaefli 
Please Review

File: prisma/schema.prisma [hunk 1]
```diff
@@ -216,17 +219,28 @@ model UserProposalFeedback {
...
+  updatedAt DateTime @default(now()) @updatedAt
+}
+
+model Responsible {
+  id String @id @unique @default(uuid())
+
+  name  String
+  email String @unique
+
+  UserProposalSupervisions UserProposalSupervision[]
```
Comment (rschlaefli) [inline prisma/schema.prisma:242]:
```suggestion
  supervisions UserProposalSupervision[]
```

File: prisma/schema.prisma [hunk 2]
```diff
@@ -216,17 +219,28 @@ model UserProposalFeedback {
...
-  supervisor String?
-  student String?
-  title String?
-  type String?
-  status String?
+  proposal   Proposal @relation(fields: [proposalId], references: [id], onDelete: Cascade)
+  proposalId String
+
+  mailReceived     String?
+  status           String?
```
Comment (rschlaefli) [inline prisma/schema.prisma:226]: use an enum with predefined status options (depending on the admin people but we should define a few steps and ask them for feedback)

File: prisma/schema.prisma [hunk 3]
```diff
@@ -143,6 +143,7 @@ model Proposal {
   supervisedBy      UserProposalSupervision[]
   applications      ProposalApplication[]
   receivedFeedbacks UserProposalFeedback[]
+  AdminInfo         AdminInfo[]
```
Comment (rschlaefli) [inline prisma/schema.prisma:146]: this is a one to one relation -> one proposal can have only one admin info associated. so no []

Comment (mxmlnwbr) [reply prisma/schema.prisma:146]: changed to AdminInfo? (unrequired)

**Comment by mxmlnwbr** [comment]
@rschlaefli 
Review Please 💯

**Comment by mxmlnwbr** [comment]
Screenshot 2024-03-07 at 09 06 03

**Comment by mxmlnwbr** [comment]
@rschlaefli 
At Least 1 approving required ;)
Please approve - if it looks good to you!

**Review by rschlaefli — APPROVED** [review]

**Pull request merged**